samples/openthread/coprocessor/README.rst

```diff
@@ -111,7 +111,7 @@ The following snippets are available:
 * ``usb`` - Enables emulating a serial port over USB for Spinel communication with the host.
 
   .. note::
-     The ``usb`` snippet is not supported for the ``nrf54l15dk/nrf54l15/cpuapp`` board target.
+     The ``usb`` snippet is not supported for the ``nrf54l15dk/nrf54l15/cpuapp`` and the ``nrf54l15dk/nrf54l05/cpuapp``board targets.
```

Comment:
```suggestion
     The ``usb`` snippet is not supported for the ``nrf54l15dk/nrf54l15/cpuapp`` and the ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
```
To fix the doc build error.

Reply:
fixed similar as in cli sample to align with @annwoj 's comment